experimental/generation/generator/packages/library/package.json

```diff
@@ -5,13 +5,13 @@
   "author": "Microsoft",
   "bugs": "https://github.com/Microsoft/BotBuilder-Samples/issues",
   "dependencies": {
-    "@microsoft/bf-lu": "^4.10.0-preview.130225",
-    "adaptive-expressions": "^4.9.0",
+    "@microsoft/bf-lu": "^4.10.0-preview.148066",
+    "adaptive-expressions": "^4.10.0-preview-148939",
```

Comment:
Interesting there is a difference in naming patterns:  preview- vs preview.

Reply:
They changed that in the build system.